Game/Resources/Shaders/lightmap.fs

```diff
@@ -0,0 +1,218 @@
+#version 430 core
```

Comment:
I don't find the difference between this shader and blinn phong

Reply:
Lightmap.fs calculates with fragment_normal or just a normal if there's no material, when this is erased from blinn_phong.fs, however i'm not sure what's the difference, regarding this.

Reply:
Yeah, Lightmap.fs just ignore all the lights sources but the one coming from the lightmap